### ISSUE TITLE
Enabling OverlayInteraction does not prevent click events from being blocked

### ISSUE THREAD

**Issue by Wisdawg** [fatihsolhan/v-onboarding#100]
**The bug**
Setting the `options` value of `preventOverlayInteraction` to `false` allows interaction but doesn't allow the actual click events to occur. This seems to be a bug.

**To Reproduce**
eg.
If you instantiate a ` <VOnboardingWrapper :options="options">` with the options below when the Tour popover appears, other buttons on the screen will be hoverable and clickable, but their actual click events won't fire. ie. Hyperlinks will not route you to a new page.

```
    const options = {
      overlay: {
        enabled: false,
        preventOverlayInteraction: true,
      },
      hideButtons: {
        previous: false,
        next: true,
        exit: false,
      },
    }
```

**Screenshots**
It looks like the FocusTrap library is being called on to prevent event propagation:
https://github.com/fatihsolhan/v-onboarding/blob/main/src/components/VOnboardingStep.vue#L96
![Screenshot 2024-05-01 at 4 41 41 PM](https://github.com/fatihsolhan/v-onboarding/assets/13368228/9bd54246-3271-4d30-9439-7b23a92ec655)

**Expected behavior**
When `preventOverlayInteraction` is made `false` interaction AND events become unblocked. Alternatively, if you want to keep interaction blocking separate from event blocking you could introduce a new value in options, perhaps called preventOverlayClickEvents? And if this feature already exists please let me know how to use it.

**Desktop (please complete the following information):**
 - OS: macOS Sonoma 14.2
 - Browser: Chrome

**Comment by seomikewaltman** [comment]
I think you need to have a tabbable element in the `VOnboardingStep`. I was having this same issue because I switched out the <button> with a <a>

**Comment by fatihsolhan** [comment]
Hi @Wisdawg! 👋

This issue has been resolved and included in v2.9.0.

Feel free to reopen if you encounter any issues. Thanks for your patience! 🙌
